topic_name: /raw_scan
sensor_frame: base_laser
